src/rpdk/core/data/schema/provider.definition.schema.v1.json

```diff
@@ -71,6 +71,17 @@
                 }
             }
         },
+        "replacementStrategy": {
+            "type": "string",
+            "description": "The valid replacement strategies are [create_then_delete] and [delete_then_create]. All other inputs are invalid.",
+            "default": [
```

Comment:
Should this be "default": "create_then_delete" (not an array)? 

I am developing a new resource type that could use this property and am looking forward to this new feature!

Reply:
Thanks a lot for the good catch! Fixing it.

Reply:
No problem. I wasn't referring to the description though, the default value is still an array:
"default": ["create_then_delete"]     ----> "default": "create_then_delete"

Reply:
Fixed that. Thanks!